qtglean/metrics.yaml

```diff
@@ -24,6 +24,7 @@ performance:
     expires: never
     time_unit: microsecond
 
+settings:
```

Comment:
How is this change related?

Reply:
It was part of the "ensure this gets updated in every PR so it doesn't get to production without being in the `setting` collection" plan. I've merged in `main`, so this no longer is changed in this PR.